plugins/modules/remote_management/lxca/lxca_cmms.py

```diff
@@ -20,6 +20,7 @@
   uuid:
     description:
       uuid of device, this is string with length greater than 16.
+    type: str
```

Comment:
same as for `lxca_nodes.py`, the options are not defined as strings in the `INPUT_ARG_SPEC ` dict. Though the tests are green
@felixfontein any ideas?

Reply:
discussed